src/wallet/rpcwallet.cpp

```diff
@@ -967,10 +969,11 @@ static UniValue addmultisigaddress(const JSONRPCRequest& request)
                     {"address_type", RPCArg::Type::STR, /* default */ "set by -addresstype", "The address type to use. Options are \"legacy\", \"p2sh-segwit\", and \"bech32\"."},
                 },
                 RPCResult{
-            "{\n"
-            "  \"address\":\"multisigaddress\",    (string) The value of the new multisig address.\n"
-            "  \"redeemScript\":\"script\"         (string) The string value of the hex-encoded redemption script.\n"
-            "}\n"
+                    RPCResult::Type::OBJ, "", "",
```

Comment:
It might be worthwhile to add a constructor where these two empty strings aren't needed. It's quite confusing as a reader to see these everywhere.

Reply:
The first one is the name and it is only used when it is needed to name it (i.e. as a key in a dictionary). Not sure if it makes sense to omit it from the constructor when it is not needed. I'll wait for others to chime in.

The second string is the description, which should never be empty. Or at least it shouldn't be made easy to leave empty. I prefer if new code is explicit about not providing a description.